setup.py

```diff
@@ -22,36 +22,38 @@
 from distutils.extension import Extension
 
 
-SRC_DIR = 'linking'
+SRC_DIR = 'primo/linking'
 
 
-if glob.glob(os.path.join(SRC_DIR, '*.pyx')):
-  try:
-    from Cython.Distutils import build_ext
-    use_cython = True
-    #from Cython.Compiler import Options
-    #Options.annotate = True
-    # Options.directive_defaults['profile'] = True
-  except ImportError:
+if glob.glob(os.path.join(SRC_DIR, '*.c')):
     use_cython = False
```

Comment:
Could you please keep indent to two spaces to be consistent with the rest of the project?